From a7a4866a31e6945c8717d71dd9cd2fb367517f5a Mon Sep 17 00:00:00 2001
From: Marc Mutz <marc.mutz@qt.io>
Date: Fri, 26 Sep 2025 10:33:17 +0200
Subject: [PATCH 415/553] Mark remaining corelib/text/*.qdoc files are
 insignificant

This completes the security marking for this subdirectory:

  $ for i in $(find src/corelib/text -type f) ; do
      if ! grep -qE '^ *(//|#) .*Qt-Security +score:' "$i"; then
          echo "$i"
      fi
  done
  src/corelib/text/qt_attribution.json

The JSON file was also explicitly marked in a previous commit.

QUIP: 23
Fixes: QTBUG-135195
Pick-to: 6.8
Change-Id: Ia780a061541aa7f16d2b7303d91d747b31ccd2fb
Reviewed-by: Matthias Rauter <matthias.rauter@qt.io>
(cherry picked from commit 6689a8750ef94e6d1014965aae11adcbf3860b5f)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/text/qlocale.qdoc    | 1 +
 src/corelib/text/qtliterals.qdoc | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/corelib/text/qlocale.qdoc b/src/corelib/text/qlocale.qdoc
index 3980e9d9a6d..bc88b27477d 100644
--- a/src/corelib/text/qlocale.qdoc
+++ b/src/corelib/text/qlocale.qdoc
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GFDL-1.3-no-invariants-only
+// Qt-Security score:insignificant reason:docs
 
 /*!
     \class QLocale
diff --git a/src/corelib/text/qtliterals.qdoc b/src/corelib/text/qtliterals.qdoc
index c4671415ee4..06ce31848f7 100644
--- a/src/corelib/text/qtliterals.qdoc
+++ b/src/corelib/text/qtliterals.qdoc
@@ -1,5 +1,6 @@
 // Copyright (C) 2021 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GFDL-1.3-no-invariants-only
+// Qt-Security score:insignificant reason:docs
 
 /*!
     \namespace QtLiterals
-- 
2.51.2

